### DISCUSSION TITLE
Teaching Tip Not Visible on Start of UWP Windows App

### DISCUSSION THREAD

**Discussion by saurabhbh21** [microsoft/microsoft-ui-xaml#7443] [Q&A]
I added a teaching tip in windows UWP app and set the default value to **TRUE**. The teaching tip is not visible when the application loads. I tested with loaded property, the Teaching Tip is getting loaded.
But, when I switched out to another application and then switch again to Windows UWP app, the "Teaching Tip" gets **Visible.**

I'm attaching the the xaml component of Teaching Tip (for reference).  

It would be really helpful if someone can help if I'm missing something or is this a bug in Teaching Tip?

```
<muxc:TeachingTip
            x:Name="TestTaskTip"
            Title="Teaching Tip: Title"
            Subtitle="Subtitle Text"
            Target="{x:Bind FontIcon}"
            IsOpen="True"
            PreferredPlacement="TopRight"
            PlacementMargin ="0,20,0,0"
            IsLightDismissEnabled="False"
            ShouldConstrainToRootBounds="False"
            Style="{StaticResource CustomTeachingTipStyle}"
            Background="{ThemeResource BackgroundBrush}"
            Foreground="{ThemeResource ForegroundBrush}"
            Loaded="TestTip_Loaded"
            IsTabStop="False">
            <Grid>
                <Grid.RowDefinitions>
                    <RowDefinition Height="*"/>
                </Grid.RowDefinitions>
                <Border
                    CornerRadius="4"
                    Grid.Row="0"
                    Margin="0,8,0,0"
                    HorizontalAlignment="Right" >
                    <Button
                        x:Name="TestButton"
                        Style="{ThemeResource ThemedButton}" 
                        FontSize="14"
                        Padding="20,6"
                        Content="Close"
                        Click="Test_Click"
                        Loaded="TestButton_Loaded"/>
                </Border>
            </Grid>
        </muxc:TeachingTip>
```

**Comment by StephenLPeters** [comment]
@saurabhbh21 do you get a loaded event in your TestButton as well as your teaching tip?  I wonder if somehow the main window is drawing on top of the teaching tip.   If you apply a large margin to the teaching tip and make the window small (something to make it so if this was the case you would see it poking out) are you able to see the tip on first load?

**Comment by saurabhbh21** [comment]
@StephenLPeters I've tested the loaded event for both (TeachingTip and TestButton). Both are working fine. 
I've done some more tests and figured out that reason you've pointed out is actually correct. Also, resizing the App window will make the TeachingTip visible. The Main Window is overriding the Teaching Tip, but the other elements (FontIcon and Textbox) which are in same root level as Teaching Tip are properly visible. 

I've tried setting margin for TeachingTip from 20 to 50. But it didn't work. I'm not sure if I completely understand you proposed solution.  Can you please explain in little detail.
